_clanky/vývoj-počtu-cizinců-v-čr-mezi-lety-2009-2018.md

```diff
@@ -100,10 +101,10 @@ Závěrečnou editaci mapy provedeme pomocí tlačítka "nové tiskové rozvrže
     description="Přidání názvu, legendy a měřítka"
 %}
 
-## Další užití, závěr
+## Další využití dat/vizualizací a zdrojů
```

Comment:
Proč tam dáváš lomítko? Do nadpisu se nedává. Vymysli jiný název nebo tam dej jen Další možnosti využití.